executor/prepared.go

```diff
@@ -136,10 +136,48 @@ func (e *PrepareExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		return ErrPrepareMulti
 	}
 	stmt := stmts[0]
+
+	//Try to handle the enable_plan_cache(bool) hint
+	//If there are multiple hints, use the last one, and throw a warning
+	planCacheHit := true
+	hitCount := 0
+	switch stmt.(type) {
```

Comment:
Only need to do these checks when `plannercore.PreparedPlanCacheEnabled()` is `true`. This check on the common code path might hurt the overall performance.